src/components/Form/SignInUp/validationConstants.ts

```diff
@@ -0,0 +1,63 @@
+export const Required = {
+  required: {
```

Comment:
```suggestion
  required: 'Required to fill.'
```

Besides I would suggest to create one single object 
```
const validationRules = {
  required: 'Required',
  email: {
    pattern: { }
  },
  phone: {
    pattern: { }
  },
// etc
}
```